### PR TITLE
FE apps: Enable specifying node version from .nvmrc, add pnpm_version

### PR DESCRIPTION
**Pull request by diasdm**
In frontend-workflow.yml:
* allow to specify the version of node using a .nvmrc file
* specify the version of pnpm installed
* enable sonarcloud

Tested in https://github.com/MOV-AI/frontend-npm-ide/actions/runs/9793603165

### CODE REVIEW

**Review by quirinpa — APPROVED** [review]

File: .github/workflows/frontend-workflow.yml [hunk 1]
```diff
@@ -74,16 +85,24 @@ jobs:
...
+      if: ${{ inputs.use_nvmrc == 'true' }}
+      with:
+        node-version-file: '.nvmrc'
+        registry-url: 'https://npm.pkg.github.com/mov-ai'
+
+    - name: Install pnpm and setup .npmrc
       if: ${{ inputs.pm == 'pnpm' }}
       run: |
-        npm install -g pnpm
+        npm install -g pnpm@${{ inputs.pnpm_version }}
```
Comment (AlexFernandes-MOVAI) [inline .github/workflows/frontend-workflow.yml:105]: what happen if `inputs.use_nvmrc == 'true'` ? `inputs.pnpm_version` is not relevant anymore, isn't ?

File: .github/workflows/frontend-workflow.yml [hunk 2]
```diff
@@ -235,16 +253,24 @@ jobs:
...
```
Comment (AlexFernandes-MOVAI) [inline .github/workflows/frontend-workflow.yml:273]: same here, I have a doubt ...

**Review by AlexFernandes-MOVAI — CHANGES_REQUESTED** [review]
please clarify the logic behind the inputs:
- inputs.pnpm_version vs inputs.use_nvmr

File: .github/workflows/frontend-workflow.yml (hunk 1 above)
Comment (quirinpa) [reply .github/workflows/frontend-workflow.yml:105]: That just chooses the node version. Not the pnpm version

Comment (diasdm) [reply .github/workflows/frontend-workflow.yml:105]: `inputs.use_nvmrc` and `inputs.pnpm_version` are independent. The first makes the action use the file `.nvmrc` to get the node version, the second is the pnpm version

**Review by AlexFernandes-MOVAI — APPROVED** [review]

**Pull request merged**